bundles/org.openhab.binding.hdpowerview/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -58,4 +82,10 @@
 		<description>Activates the scene</description>
 	</channel-type>
 
+	<channel-type id="refresh">
+		<item-type>Switch</item-type>
+		<label>Refresh Cache</label>
+		<description>Instructs the Hub to refresh its Shade data cache</description>
+	</channel-type>
```

Comment:
Is this the Channel you wanted to remove?

Reply:
Yes. Removed.